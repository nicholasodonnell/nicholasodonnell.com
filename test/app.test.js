import React from 'react'
import { render } from '@testing-library/react'

import App from '../src/app'

jest.mock('react-tsparticles', () => () => <canvas />)

describe('<App />', () => {
  it('renders', () => {
    const app = render(
      <App />
    )

    expect(app.baseElement).toMatchSnapshot()
  })
})
