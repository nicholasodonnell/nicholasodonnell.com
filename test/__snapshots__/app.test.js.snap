// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders 1`] = `
<body>
  <div>
    <main
      class="sc-iBkjds klUzpK"
    >
      <section
        class="sc-hKMtZM dMoMas"
      >
        <h1
          class="sc-eCYdqJ tumFC"
        >
          Hello, I'm
           
          <span
            class="sc-gsnTZi iTabn"
            data-text="NICHOLAS O'DONNELL"
          >
            Nicholas O'Donnell
          </span>
        </h1>
        <h2
          class="sc-jSMfEi faRZsE"
        >
          I'm a full-stack senior software engineer
        </h2>
      </section>
      <section
        class="sc-dkzDqf kGYhMy"
      >
        <a
          class="sc-bczRLJ jKCSZy"
          href="https://github.com/nicholasodonnell"
          target="_blank"
        >
          Explore Work
        </a>
        <a
          class="sc-bczRLJ jKCSZy"
          href="https://www.linkedin.com/in/nicholas-odonnell"
          target="_blank"
        >
          View Profile
        </a>
      </section>
      <canvas />
    </main>
  </div>
</body>
`;
